### ISSUE TITLE
CBZ processing error

### ISSUE THREAD

**Issue by psquiddy** [rumesh986/comicconvert#6]
Line 55 currently reads:
`img2pdf -o "$pdffile" .cbr/*.jpg
`
Should be corrected to:
`img2pdf -o "$pdffile" .cbz/*.jpg
`

**Comment by riadabdel** [comment]
thx dude you solved my issue : 
img2pdf: error: argument infile: ".cbr/*.jpg" does not exist

**Comment by jfburdet** [comment]
Yeah, there is a typo in the script file, at line 56 wich should be : "img2pdf -o "$pdffile" .cbz/*.jpg"

**Comment by rumesh986** [comment]
This should be fixed in the latest commit
